File: round 1/cobra/problemtest.c
#include<stdio.h>
int main()
{ int r,n,sum=0;
  scanf("%d",&n);
  do
{
  while(n>9);
  {r=n%10;
   sum=sum+r;
   n=n/10;
}
n=sum;
sum=0;
}while(n>=10);
if(n==1)
{ 
 printf("yes");
}
else
{ printf("no");
}
  return 0;
}

File: round 1/RedandBlue/problemtest.java
import java.util.*;
class problem4
{
public static void main()
{
Scanner sc=new Scanner(System.in);
int s=0, sum=0;
while(n>0)
{
p=n%10;
s=s+p;
n=n/10;
}
if(s==1)
println("yes")
elseif(s>9)
{
x=s%10;
sum=sum+s;
if(sum==1)
println("yes");
else
println("no");
}
}
}
